tasks/pipeline-osconf.yml

```diff
@@ -56,6 +56,18 @@
     state: "link"
   when: "create_shareddir and archivematica_src_shareddir != '/var/archivematica/sharedDirectory'"
 
+# check if clamav is running with file or tcp socket
+- name: "Check if clamav is running with file or tcp socket"
```

Comment:
This will default to using the file socket, although tcp socket could be also enabled, is that what we want?

Reply:
Hi Santi, it detects whether the file or tcp socket is used. When using https://github.com/artefactual-labs/ansible-clamav/pull/6 there is no option to use the TCP and UNIX socket simultaneously